go-controller/pkg/node/iptables/iptables_manager.go

```diff
@@ -14,13 +14,33 @@ import (
 	kexec "k8s.io/utils/exec"
 )
 
-// Chain allows users to manage rules for a particular iptables table and chain for ipv4 and ipv6 protocols
-type Chain struct {
+// rulesIndex structure is used as a golang map key to point to a set of IPTable rules. It holds all the necessary info
+// to retrieve IPTable rules
+type rulesIndex struct {
 	Table iptables.Table
 	Chain iptables.Chain
 	Proto iptables.Protocol
 }
 
+// rules represents one or more rules within a chain of a specific IP version
+type rules struct {
+	exclusive bool
```

Comment:
whats exclusive mean? ovnk fully owns these rules?

Reply:
Added a comment above it. Let me know if it makes sense.